2nd review: the two items from the 1st review are both properly addressed (`Related Issues` is now exempted from the `None/N/A` fallback, and a StrictMode regression test was added). However, lifting the controller up to `OptionsPage` introduced a **regression that breaks the "add a second account" path entirely.** Since that is the happy path for multi-account users, I'm treating it as a blocker.

## Blocker: panel gets stuck after the first successful connect

The guard in [entrypoints/options/options-page.tsx:44-49](entrypoints/options/options-page.tsx) only calls `controller.start()` when `phase === "idle"`:

```tsx
const openAddPanel = () => {
  setShowAddPanel(true);
  if (controller.state.phase === "idle") {
    controller.start();
  }
};
```

Because the controller now lives on the parent, terminal state from a completed flow (`connected`) survives across panel opens. What actually happens:

1. First account connects successfully → `onConnected` fires → `setShowAddPanel(false)` → controller stays at `{phase: "connected", accountId: "pending"}`.
2. User clicks "+ Add another account" again.
3. `phase !== "idle"` → `start()` is skipped.
4. Panel mounts with `state.phase === "connected"` and renders "Account connected. You can add another account or close this panel." with **no interactive path forward — dead end.**

Before the lift, each `AddAccountPanel` unmount/remount produced a fresh controller at `phase: "idle"`, which is why this path worked previously.

### Repro (against PR head, locally)

```ts
// after first add succeeds
initiateDeviceFlow: spy called 1 time
// click "+ Add another account" a second time
expect(initiateDeviceFlow).toHaveBeenCalledTimes(2);
// → AssertionError: expected "spy" to be called 2 times, but got 1 times
```

`expired` and `denied` have the same structural problem on re-open, but the current UI has no way to close the panel from those phases, so `connected` is the only one observable in practice today.

### Suggested fix

Narrow the guard so it only blocks **active** phases, or just always call `start()` from the click handler (React does not double-invoke event handlers under StrictMode, so this would not resurrect the original regression):

```tsx
const openAddPanel = () => {
  setShowAddPanel(true);
  const inFlight =
    controller.state.phase === "initiating" ||
    controller.state.phase === "waiting" ||
    controller.state.phase === "fetching_installations";
  if (!inFlight) {
    controller.start();
  }
};
```

This cleanly resets from any terminal phase (`connected` / `expired` / `denied` / `fatal`) on re-open. Please add a test that asserts a second `initiateDeviceFlow` call after a successful first connect. It would also be worth pinning down the re-auth-while-panel-open interaction (`AccountsList` → `onReauthenticate`), since it runs through the same guard.

## Non-blocking

None. The documentation changes (AGENTS.md Pull Request Policy section, expanded pr-guideline, PR template, and the one-line manual-chrome-testing update) are all internally consistent and resolve the prior review concern.

## Validation (on PR head `eb78cf6`)

- `pnpm lint`: pass
- `pnpm typecheck`: pass
- `pnpm test`: 81/81 pass — but no test covers the second-add regression above, which is how it slipped through.

Please fix the blocker and request another review.

--- a/entrypoints/options/options-page.tsx
+++ b/entrypoints/options/options-page.tsx
@@ -39,7 +39,11 @@ export function OptionsPage() {
 
   const openAddPanel = () => {
     setShowAddPanel(true);
-    if (controller.state.phase === "idle") {
+    const inFlight =
+      controller.state.phase === "initiating" ||
+      controller.state.phase === "waiting" ||
+      controller.state.phase === "fetching_installations";
+    if (!inFlight) {
       controller.start();
     }
   };

--- a/tests/options-page.test.ts
+++ b/tests/options-page.test.ts
@@ -218,6 +218,63 @@ describe("OptionsPage", () => {
     ).not.toBeNull();
   });
 
+  it("restarts the device flow when reopening the add-account panel after a successful connection", async () => {
+    await renderOptionsPage();
+
+    const auth = await import("../src/github/auth");
+    const initiateDeviceFlow =
+      auth.initiateDeviceFlow as unknown as ReturnType<typeof vi.fn>;
+    const pollForAccessToken =
+      auth.pollForAccessToken as unknown as ReturnType<typeof vi.fn>;
+    const fetchAuthenticatedUser =
+      auth.fetchAuthenticatedUser as unknown as ReturnType<typeof vi.fn>;
+    const fetchUserInstallations =
+      auth.fetchUserInstallations as unknown as ReturnType<typeof vi.fn>;
+
+    initiateDeviceFlow.mockResolvedValue({
+      deviceCode: "dc",
+      userCode: "ABCD-EFGH",
+      verificationUri: "https://github.com/login/device",
+      verificationUriComplete:
+        "https://github.com/login/device?user_code=ABCD-EFGH",
+      expiresIn: 900,
+      interval: 0,
+    });
+    pollForAccessToken.mockResolvedValue({
+      status: "success",
+      accessToken: "ghu_abc",
+    });
+    fetchAuthenticatedUser.mockResolvedValue({
+      login: "hon454",
+      avatarUrl: null,
+    });
+    fetchUserInstallations.mockResolvedValue([]);
+
+    await act(async () => {
+      document
+        .querySelector<HTMLButtonElement>('[data-testid="accounts-add"]')!
+        .click();
+      await Promise.resolve();
+      await new Promise((resolve) => setTimeout(resolve, 0));
+      await Promise.resolve();
+      await new Promise((resolve) => setTimeout(resolve, 0));
+    });
+
+    expect(initiateDeviceFlow).toHaveBeenCalledTimes(1);
+    expect(
+      document.querySelector('[data-testid="accounts-add"]'),
+    ).not.toBeNull();
+
+    await act(async () => {
+      document
+        .querySelector<HTMLButtonElement>('[data-testid="accounts-add"]')!
+        .click();
+      await Promise.resolve();
+    });
+
+    expect(initiateDeviceFlow).toHaveBeenCalledTimes(2);
+  });
+
   it("does not start the device flow twice under React StrictMode", async () => {
     await renderOptionsPageInStrictMode();
 

